### PR TITLE
ADAPTSM-74 | @rebeccahongsf | fixup form styles

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Adjust `selected` text for Payment options
- Add select field focus state
- Add cursor pointer for radio and checkboxes
- Update hardcoded color to cardinal red

# Review By (Date)
- asap
# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://deploy-preview-671--stanford-alumni.netlify.app/membership/join
2. Verify that when selecting either full payment or installment, the ✔️ Selected text appears the same within the cards
3. Continue to form
4. Verify that upon tabbing to the select list, the gradient outline appears
5. Verify that upon hovering over the radio and checkboxes, the cursor pointer appears
6. Navigate to https://deploy-preview-671--stanford-alumni.netlify.app/membership/saacard
7. Verify the color (this is dependent on the membership for the test account)
8. Review code 👾

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
Awesome - thank you for these fixes @rebeccahongsf !! 💯 

Please feel free to merge this PR as is. I found a bonus item that's low priority, but I'll leave it up to you if you want to fix it or not (I think it might be a quick fix because I recall that you fixed a similar issue with other divs)

Looks like the border gradient is a bit strange for these input elements (text fields and select). Seems like it's trying to be a gradient but it's not a smooth gradient? 😂 

Screenshot 2023-05-18 at 1 50 30 PM

File: src/components/cards/membershipCard.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const MembershipCard = ({ blok: { publicCtaGroup, ctaGroup }, blok }) => {
       ) {
         fetchImages(true, SaaLogoWhite, SaaCardBg);
         setBgColor(
-          'su-bg-gradient-to-b su-from-[#8E1515] su-to-digital-red-light'
+          'su-bg-gradient-to-b su-from-cardinal-red su-to-digital-red-light'
```
Comment (yvonnetangsu) [inline src/components/cards/membershipCard.js:41]: YES!!

File: src/styles/forms.css [hunk 2]
```diff
@@ -113,10 +113,9 @@
 .ggeWidget .ggeQuestion input[type="number"]:focus,
 .ggeWidget .ggeQuestion input[type="text"]:focus,
 .ggeWidget .ggeQuestion input[type="date"]:focus,
+.ggeWidget .ggeQuestion select:focus,
 .ggeWidget .ggeQuestion textarea:focus {
-  @apply su-border-2 su-border-transparent;
-  border-image-source: linear-gradient(0deg, #017e7c 84.05%, #505eec 15.6%);
-  border-image-slice: 1;
+  @apply su-border-2 su-border-transparent su-gradient-border su-border-to-rt-palo-verde-dark-to-saa-electric-blue;
```
Comment (rebeccahongsf) [inline src/styles/forms.css:118]: @yvonnetangsu I updated the fields focus state to use our TW gradient border!

Comment (yvonnetangsu) [reply src/styles/forms.css:118]: Looks so much better now - thank you for this update! 🥇

**Pull request merged**